build-system/tasks/presubmit-checks.js

```diff
@@ -36,7 +36,13 @@ var forbiddenTerms = {
   'it\\.only': '',
   'console\\.\\w+\\(': 'If you run against this, use console/*OK*/.log to ' +
       'whitelist a legit case.',
-  'cookie\\W': requiresReviewPrivacy,
+  'cookie\\W': {
+    message: requiresReviewPrivacy,
+    whitelist: [
+      'src/experiments.js',
```

Comment:
So, I'm changing implementation a bit. So, could you please submit this with empty `whitelist: []`?